packages/shared/src/components/onboarding/OnboardingHeader.tsx

```diff
@@ -11,14 +11,16 @@ import { OnboardingStep, wrapperMaxWidth } from './common';
 type OnboardingHeaderProps = {
   showOnboardingPage: boolean;
   setAuth: Dispatch<SetStateAction<AuthProps>>;
-  onClickNext: () => void;
+  onClickCreateFeed: () => void;
+  onClickNext?: () => void;
```

Comment:
Same already had some usecases so didn't want to disrupt too much.

Reply:
wouldn't they do the same thing?